doc/hacking.md

```diff
@@ -27,19 +27,16 @@ we're bussy), see contact information in the main README.md of this project.
 # Naming and Glossary
 
 
-* Virtual-IP - the IP that someone has assigned as result of using Galaxy42 network.
-It often will be in form of IPv6, from some "reserved" block (that is at least unlikelly to be assigned by IANA as part of
-the "regular" Internet).
+* Hash-IP - the IP that someone can seldomly own without requiring permission from any authority, because it is derived from public key with some form of Hash.
```

Comment:
Maybe:?

Hash-IP Address: A Virtual IP Address based upon a cryptographic hash. Every Hash-IP is derived from a Public Key. Use of any Hash-IP demands access to the associated Private Key (for that particular Hash-IP Address).